Project.toml

```diff
@@ -3,13 +3,13 @@ uuid = "b4f28e30-c73f-5eaf-a395-8a9db949a742"
 version = "1.2.0"
```

Comment:
Will make sense to move the version to "1.3.0"?

Reply:
Yes